lte/gateway/c/oai/tasks/s1ap/s1ap_mme.c

```diff
@@ -577,6 +555,7 @@ void s1ap_remove_ue(s1ap_state_t* state, ue_description_t* ue_ref) {
       s1ap_imsi_map->mme_ue_id_imsi_htbl, (const hash_key_t) mme_ue_s1ap_id,
       &imsi64);
   delete_s1ap_ue_state(imsi64);
+  hashtable_uint64_ts_free(s1ap_imsi_map->mme_ue_id_imsi_htbl, mme_ue_s1ap_id);
```

Comment:
what is the difference between free vs remove?

Reply:
same question came to me, after investigating:
- for `hashtable_uint64` implementation, there is no difference between the `ts_free` and `ts_remove` handlers, both of them call `free_wrapper` on the node to be removed, and update the nodes accordingly
- for `hashtable`, `ts_free` frees the object on the hashtable, and `ts_remove` removes the object on the hashtable nodes, but does not free the object itself

Reply:
I see that hashtable_uint64_ts_remove used everywhere except for one. Let's eliminate the usage of hashtable_uint64_ts_free as it would lead to confusion. on a separate PR maybe we should revisit hashtable itself and use a better naming than remove if it is not freeing but simply popping the entry.

Reply:
@ulaskozat Removed `hashtable_uint64_ts_free` usage and function